Documentation/ColumnsConfig/Properties/GroupInternalType.rst.txt

```diff
@@ -25,6 +17,3 @@
       This will create a field where database records can be attached as references.
 
     The default value is none of them - you must specify one of the values correctly!
-
-    .. important::
-        Types "file" and "file\_reference" should not be used anymore. Use FAL references instead.
```

Comment:
I would leave it in and change the wording: "have been removed" and add some information about possible migration paths for ppl who are updating.